Model/Customer.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * @author Nedelin Slavov <ned@metrilo.com>
+ */
+
+namespace Metrilo\Analytics\Model;
+
+/**
+ * Model getting orders by chunks for Metrilo import
+ *
+ * @author Nedelin Slavov <ned@metrilo.com>
+ */
+class Customer
+{
+    private $ordersTotal    = 0;
+    private $totalChunks    = 0;
+    private $chunkItems     = 15;
+    private $customersArray = [];
+
+    public function __construct(
+        \Magento\Customer\Model\ResourceModel\Customer\CollectionFactory $customerCollection,
+        \Magento\Newsletter\Model\Subscriber $subscriber
+    ) {
+        $this->customerCollection = $customerCollection;
+        $this->subscriber         = $subscriber;
+    }
+
+    /**
+     * Get chunk customer data for import
+     *
+     * @param  int
+     * @return mixed
+     */
+    public function getCustomers($storeId, $chunkId)
```

Comment:
How are these chunks going to work? Isn't there an option to have more order chunks than customer chunks?